sdk/core/core-xml/test/xml.spec.ts

```diff
@@ -340,11 +438,11 @@ describe("XML serializer", function() {
       );
     });
 
-    it("with element with child undefined element", async function() {
-      const xml = await stringifyXML(
+    it("with element with child empty element", async function() {
+      const xml = stringifyXML(
         {
           fruit: {
-            apples: undefined
```

Comment:
This is the only change I had to do to the existing tests. FXP removes a node if its value is `undefined` which I think aligns with the JS concept of `undefined`. xml2js and DOM would make it an empty node <foo />.

Are you aware of any implications this change may have?